### ISSUE TITLE
PLUMED: I cannot understand line: PAIRENTROPY LABEL=s2 ATOMS=1-256 MAXR=0.7 SIGMA=0.0125 NLIST NL_CUTOFF=0.8 NL_STRIDE=4 PLUMED: Maybe a missing space or a typo?

### ISSUE THREAD

**Issue by nirzaa** [plumed/plumed2#766]
I wanted to use the CV PAIRENTROPY which needs the files as attached here:
(*) https://sites.google.com/site/pablompiaggi/scripts/pair-entropy-cv?authuser=0
I made a new folder named PAIRENTROPY inside my src/ folder and copy to that folder the 3 files that were noted at (*).
After that I ran: `make yes-PAIRENTROPY`
It told me that: `Installing package PAIRENTROPY`
Then I compiled using: `make -j 6 g++_openmpi`

But after that, when I ran the code again, it back and told me the same error message:
`PLUMED: ERROR
PLUMED: I cannot understand line: PAIRENTROPY LABEL=s2 ATOMS=1-256 MAXR=0.7 SIGMA=0.0125 NLIST NL_CUTOFF=0.8 NL_STRIDE=4
PLUMED: Maybe a missing space or a typo?`

Any idea what I have done wrong?

Thank you very much in advance

**Comment by PabloPiaggi** [comment]
I will get back to you on this as soon as possible by email. Thanks.

**Comment by PabloPiaggi** [comment]
Hello @nirzaa , I simplified the procedure to use the pair entropy CV. I have uploaded examples to [this repository](https://github.com/PabloPiaggi/PRL-2017-PairEntropy) and it has an associated Plumed Nest egg [here](https://www.plumed-nest.org/eggs/21/047/). You should be able to run simulations using this CV without making changes to the source code. Please let me know if your problem is solved so this issue can be closed. 

@gtribello and @GiovanniBussi  this CV has never been added to the PLUMED repository because I don't see an appropriate place for it (colvar?, crystallization?, perhaps a miscellaneous category?). I would be happy to add it to the official plumed repo if you think it is worthwhile. Please let me know. For the time being, this feature is available using the LOAD keyword. Thanks.

**Comment by valsson** [comment]
Hello @PabloPiaggi, you could just simply add it as a single CV module, same as I did today after @GiovanniBussi suggestions, see #768, and #770 and https://github.com/plumed/plumed2/tree/master/src/s2cm, then you can also keep the copyright on your name. 

Regards,
Omar

**Comment by invemichele** [comment]
Hi Pablo, I have been gathering some of the structure factor CVs we developed with the idea of eventually creating a module, see https://github.com/invemichele/plumed2/tree/structure_factor/src/structurefactor
Since the pair entropy is strictly connected to the Debye structure factor, maybe it would be a good idea to include it as well? Or do you think it might be confusing, and prefer to make a separate module?
We can discuss this via PM

Tagging other people that are probably interested in the project @luigibonati @Tarakk @ HaiyangNiu (cannot find his github)

**Comment by gtribello** [comment]
Hello @PabloPiaggi I think a separate module is definitely the right way to include this stuff.  If you want to combine in with the stuff @invemichele is suggesting that is fine.  I wouldn't put it in colvar or crystallisation though.

**Comment by PabloPiaggi** [comment]
Thank you @valsson @invemichele and @gtribello for your suggestions. I like the idea of having a module both with g(r) and structure factor related stuff. @invemichele when are you planning to open a PR with the new module? I could contribute directly there and we can also discuss modifications to make things consistent. Thanks.

**Comment by nirzaa** [comment]
> 

Works great, thank you very much. This issue can be closed.

**Comment by GiovanniBussi** [comment]
@PabloPiaggi @invemichele @gtribello I will release 2.8beta today, in case you want to add an external module later I think it's fine though I would then suggest to add it directly to master. Up to you.

**Comment by invemichele** [comment]
I would do things without rush and add it to master before the end of the year. This way we can decide a proper name for the module, add all the regtests and write a small description of the module.

However, if @PabloPiaggi you want the pairentropy in v2.8, I can spend some time this afternoon/evening to adjust the structurefactor part of the module

**Comment by PabloPiaggi** [comment]
No, @invemichele there's no rush. Opening a PR against the master branch some time in the future sounds good to me.
